lensless/admm.py

```diff
@@ -181,6 +181,15 @@ def _X_update(self):
             self._xi + self._mu1 * self._forward_out + self._convolver._pad(self._data)
         )
 
+    def _W_update(self):
```

Comment:
nothing change in this function right?

Reply:
No, I just change the order to reflect the order in update